ultron/src/main/java/com/atiurin/ultron/custom/espresso/matcher/SuitableRootMatcher.kt

```diff
@@ -0,0 +1,35 @@
+package com.atiurin.ultron.custom.espresso.matcher
+
+import android.view.View
+import androidx.core.view.allViews
+import androidx.core.view.isVisible
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.Root
+import com.atiurin.ultron.core.espresso.UltronEspressoInteraction
+import com.atiurin.ultron.core.espresso.recyclerview.UltronRecyclerView
+import com.atiurin.ultron.extensions.withSuitableRoot
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+fun withSuitableRoot(viewMatcher: Matcher<View>): Matcher<Root> {
```

Comment:
Pls, add a javadoc with description of the method purpose. User should understand the usecase

Reply:
https://github.com/open-tool/ultron/wiki/Avoiding-nontrivial-element-lookup-exceptions